python/cusignal/windows/windows.py

```diff
@@ -302,8 +313,9 @@ def parzen(M, sym=True):
     na = cp.extract(n < -(M - 1) / 4.0, n)
     nb = cp.extract(abs(n) <= (M - 1) / 4.0, n)
     wa = 2 * (1 - abs(na) / (M / 2.0)) ** 3.0
-    wb = (1 - 6 * (abs(nb) / (M / 2.0)) ** 2.0 +
-          6 * (abs(nb) / (M / 2.0)) ** 3.0)
+    wb = (
+        1 - 6 * (abs(nb) / (M / 2.0)) ** 2.0 + 6 * (abs(nb) / (M / 2.0)) ** 3.0
```

Comment:
Is it defined somewhere why you are using the number  '6'

Reply:
We'll need to ask @awthomp. This only got flagged because of formatting changes.

Reply:
It's probably in the paper. Can I push this and then figure it out?